client/client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import requests
+import urlparse
+from django.conf import settings as django_settings
+
+
+class Client(object):
+
+    def __init__(self):
+        self._configure()
+
+    def _configure(self):
+        self.endpoint = "http://172.16.16.24:4568"
+        self.admin_uid = "1"
+        self.headers = {
+            'authorization': "Bearer {}".format(django_settings.OPENEDX_NODEBB_DISCUSSION['NODEBB_API_TOKEN']),
```

Comment:
Use proper casing for `Authorization` and `Content-Type`

Reply:
done.